client/my-sites/scan/main.tsx

```diff
@@ -55,9 +55,9 @@ interface Props {
 	};
 	applySiteOffset: applySiteOffsetType | null;
 	dispatchRecordTracksEvent: typeof recordTracksEvent;
-	dispatchScanRun: typeof triggerScanRun;
-	dispatchIncrementCounter: typeof incrementCounter;
-	dispatchSetReviewPromptValid: typeof setValidFrom;
+	dispatchScanRun: ( arg0: number ) => void;
+	dispatchIncrementCounter: ( arg0: string, arg1: boolean, arg2: boolean ) => void;
+	dispatchSetReviewPromptValid: ( arg0: 'restore' | 'scan', arg1: number | null ) => void;
```

Comment:
`typeof X` does not work here because we are calling those functions rather than just returning them. The actual interface is something like `( arg0, ..., argn ) => void`.